app/src/main/res/drawable/ic_feedback_black_24dp.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="24dp"
+        android:height="24dp"
+        android:viewportWidth="24.0"
+        android:viewportHeight="24.0">
+    <path
+        android:fillColor="#FF000000"
```

Comment:
Please don't hardcode the color but use the attribute that is being used by all the other icons, find any icon and you'll get what I mean, also rename the file once u done.

Reply:
I undertstood the renaming part, I'll rename the files as the ones you're already using, but I didn't understand what I need to change. Can you be more specific? or show me which lines I should change?

Reply:
Fixed.